HLeithner/joomla-cms
# Add prepared statements for plg_extension-finder

### Summary of Changes
Updated SQL queries to prepared statements and made some cleanups around the queries.


### Testing Instructions
This Plugin is smartsearch related it manage common words on language installation and uninstall.

* Activate smartsearch
* Install a language with common words
* check the #__finder_terms_common table if the words have been added.
* Uninstall a language with common words and check if the table get cleaned up



### Expected result
Nothing changed.
